pyoozie/tags.py

```diff
@@ -16,6 +19,9 @@
 ALLOWABLE_NAME_CHARS = set(string.ascii_letters + string.punctuation + string.digits + ' ')
 ONE_HUNDRED_YEARS = 100 * 365.24
 
+PropertyValuesType = typing.Optional[typing.Dict[typing.Text, typing.Any]]
```

Comment:
I'm not sure which iteration of Python 3.5 that it was added in, but it's in [the latest Python 3.5 docs](https://docs.python.org/3.5/library/typing.html#typing.Text). For other Python versions it comes from the [PyPi `typing` package](https://pypi.python.org/pypi/typing).

> off of this type declaration? 

Good idea 👍